paper.md

```diff
@@ -26,90 +26,69 @@ affiliations:
+In addition, Qiskit needs to traverse the graph. Some transformations, such as greedily merging instructions to reduce circuit depth, require graph traversal. _retworkx_ offers the methods for traversals such as breadth-first search, depth-first search, and topological sorting.
 
 ## Subgraph Isomorphism
 
-The third use case is to detect a pattern graph within a larger graph using the VF2 or VF2++ algorithms [@Cordella2004;@Juttner2018]. We compare the time to answer if pairs of graphs from the ARG Database are subgraph-isomorphic [@DeSanto2003]. The graphs are unlabeled, bounded-valence graphs ranging from $20$ to $1000$ nodes with valence $\upsilon \in \{3, 6, 9 \}$. They are organized in pairs such that the subgraph size is either $20 \%$, $40 \%$ or $60 \%$ of the full graph.
+The second use case is based on qubit mapping problem for Noisy Intermediate-Scale Quantum (NISQ) devices [@Bharti2021]. NISQ devices do not have full connectivity among qubits, hence Qiskit needs to take into account an undirected graph representing the connectivity of the device when compiling quantum circuits. Qiskit transforms the quantum circuit such that the pairs of qubits executing two-qubit gates respect the device's architectural constraints. There are many proposed solutions to the qubit mapping problem, including algorithms based on subgraph isomorphism [@Li2021].
+
+![Graph representing the connectivity of the **`ibmq_montreal`** device. Qiskit can check if the required connectivity by a circuit is subgraph isomorphic to the device's connectivity when solving the qubit mapping problem.\label{fig:dagexample}](paper_img/example_coupling_map.png){ width=52.5% height==52.5% }
 
-The results in Figure \ref{fig:subgraphisomorphism} show that _retworkx_ consistently outperforms _NetworkX_ by two orders of magnitude. For $n = 1000$ nodes, _retworkx_ has averaged around the order of $10^{-1}$ seconds while _NetworkX_ is closer to the order of $10^{1}$ seconds. Compared to other libraries, _retworkx_ leads the benchmark together with _igraph_, albeit _retworkx_ performs slightly better as the number of nodes grows larger.
+_retworkx_ implements the VF2 algorithm [@Cordella2004] and some of the VF2++ heuristics [@Juttner2018] to solve subgraph isomorphism. The implementations include both checking if a mapping exists and returning a mapping among the nodes. Qiskit leverages _retworkx_ to provide an experimental layout method based on VF2. Qiskit checks if the graph representing the connectvity required by the circuit and the connectivity provided by the device are subgraph isomorphic. If they are, Qiskit uses VF2 mapping to map the qubits without adding SWAP instructions to the quantum circuit. 
 
+# Limitations
```

Comment:
I would rename this section or integrate the contents here into an earlier section outlining what retworkx is. This seems out of place as a standalone section. I get what you're saying about it being important to call out in the paper to avoid potential confusion about it being a drop in replacement but having a standalone section titled limitations doesn't seem like the right fit for that intent. (especially because it really was a design decision not a limitation of the library).

Reply:
I would keep the section and call out here that there are some functionality gaps with NetworkX.

Reply:
Yeah, that works too, that would make this more about current limitations with the library (and one we'll hopefully address over time)